### PR TITLE
SED-1521 disabling the scheduler

### PR DESCRIPTION
**Pull request by neogucky**

### CODE REVIEW

File: projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { HttpClient } from '@angular/common/http';
 
 import { SchedulerService } from '../../generated';
 import { BaseHttpRequest } from '../../generated/core/BaseHttpRequest';
+import { map } from 'rxjs/operators';
+import { tap } from 'rxjs';
```
Comment (dvladir) [inline projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts:11]: No usage of these operators

File: projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts [hunk 2]
```diff
@@ -26,4 +28,8 @@ export class AugmentedSchedulerService extends SchedulerService {
     //@ts-ignore
     return this._httpClient.post<any>('rest/scheduler/task/' + id + '/execute', null, { responseType: 'text' });
   }
+
+  public isSchedulerEnabled(): Observable<boolean> {
+    return this._httpClient.get<any>('rest/settings/scheduler_enabled');
```
Comment (dvladir) [inline projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts:33]: If you know that result will be boolean, it make sense to write invocation as `this._httpClient.get<boolean>`

**Review by dvladir — DISMISSED** [review]
I've put some non critical remarks. But it's all find in common

File: projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts (hunk 2 above)
Comment (neogucky) [inline projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts:33]:
```suggestion
    return this._httpClient.get<boolean>('rest/settings/scheduler_enabled');
```

File: projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts (hunk 1 above)
Comment (neogucky) [inline projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts:11]:
```suggestion
```

**Review by dvladir — APPROVED** [review]

**Pull request merged**